config/samples/hpa.yaml

```diff
@@ -31,4 +31,4 @@ spec:
       name: cpu
       target:
         type: Utilization
-        averageUtilization: 50
+        averageUtilization: 50
```

Comment:
This change isn't related to this PR and should not be in this diff.

Reply:
I'm a little lost...It looks like nothing was changed? How do I remove a diff that effectively hasn't changed anything?

Reply:
If you look at the diff you can see a red circle with a line in it. That means there is no trailing newline at the end of the file. The change is that you've removed this trailing newline.